test/e2e/specs/all-features/record/related-table-link.spec.js

```diff
@@ -407,17 +407,12 @@ describe('View existing record,', function() {
                     expect(addRelatedRecordLink.getText()).toBe("Add", "The Add button is not displayed as Add");
 
                     addRelatedRecordLink.click().then(function(){
-                    }).then(function(){
-                        browser.wait(function () {
-                               return chaisePage.recordsetPage.getRows().count().then(function (ct) {
-                                   return (ct > 0);
-                               });
-                           });
-                        rows = chaisePage.recordsetPage.getRows();
-                    }).then(function(ct){
+                        chaisePage.waitForElementInverse(element.all(by.id("spinner")).first());
```

Comment:
I had the same failure in my branch for histograms. Instead you should remove this line and mae the other changes I suggested.